### PR TITLE
Update PR guidelines with screenshots

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description
This PR updates some PR guidelines (with screenshots) to address some pain-points I experience as a maintainer of some CC repos.

## Checklist
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [ ] I added tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin
```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

### CODE REVIEW

File: content/contributing-code/pr-guidelines/contents.lr [hunk 1]
```diff
@@ -8,20 +8,40 @@ body:
...
-    * Make a pull request as soon as you can and mark the title with a "[WIP]". You can create a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).
+    * Make a pull request as soon as you can and **mark the title with a "[WIP]"**. You can create a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).
+    <br/>
+    [Screenshot: How to create draft PR?](draft_pr.gif)
 * **Describe Your Pull Request**
-    * Use the format specified in pull request template for the repository.
+    * Use the format specified in pull request template for the repository. **Populate the stencil completely** for maximum verbosity.
+        * Tag the actual issue number by replacing `#[issue_number]` e.g. `#42`.
+          This closes the issue when your PR is merged.
+        * Tag the actual issue author by replace `@[author]` e.g. `@issue_author`.
```
[the remark this answers is not in the document]
Comment (dhruvkb) [reply content/contributing-code/pr-guidelines/contents.lr]: I'll submit a PR to https://github.com/creativecommons/.github as I do like Vocabulary's templates better.

**Comment by dhruvkb** [comment]
Full page screenshot.
image

**Comment by dhruvkb** [comment]
I was torn between including images inline versus linking to them. But then I saw other screenshots links and decided to go with the former.

File: content/contributing-code/pr-guidelines/contents.lr (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhruvkb) [reply content/contributing-code/pr-guidelines/contents.lr]: The updated, more descriptive and more verbose PR template: creativecommons/.github#8

**Comment by kgodey** [comment]
thank you! I'll review this after everyone has had a chance to review the PR template change, which will affect all repos.

File: content/contributing-code/pr-guidelines/contents.lr [hunk 2]
```diff
@@ -8,20 +8,40 @@ body:
...
+    * Make a pull request as soon as you can and **mark the title with a "[WIP]"**. You can create a [draft pull request](https://help.github.com/en/articles/about-pull-requests#draft-pull-requests).
+    <br/>
+    [Screenshot: How to create draft PR?](draft_pr.gif)
 * **Describe Your Pull Request**
-    * Use the format specified in pull request template for the repository.
+    * Use the format specified in pull request template for the repository. **Populate the stencil completely** for maximum verbosity.
+        * Tag the actual issue number by replacing `#[issue_number]` e.g. `#42`.
+          This closes the issue when your PR is merged.
+        * Tag the actual issue author by replacing `@[author]` e.g. `@issue_author`.
+          This brings the reporter of the issue into the conversation.
```
Comment (dhruvkb) [inline content/contributing-code/pr-guidelines/contents.lr]: Justification

**Pull request merged**